app/qml/components/MMBusyIndicator.qml

```diff
@@ -12,12 +12,13 @@ import QtQuick
 Item {
   id: root
 
-  property bool running: false
+  required property real size
```

Comment:
Can we have a default size? I would expect it to have one set size that I won't need to edit